catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrCatalogProvider.java

```diff
@@ -286,7 +288,7 @@ public CreateResponse create(CreateRequest request) throws IngestException {
         }
 
         pendingNrtIndex.putAll(output.stream()
-                .collect(Collectors.toMap(Metacard::getId, Function.identity())));
+                .collect(Collectors.toMap(Metacard::getId, m -> copyMetacard(m))));
```

Comment:
This defensive logic took a few tries before all tests were green.  Based on that I think there is already enough coverage that tests will fail if the defensive logic is broken.